besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1294,6 +1294,11 @@ static class PermissionsOptionGroup {
           "Specifies the maximum number of blocks to retrieve logs from via RPC. Must be >=0. 0 specifies no limit  (default: ${DEFAULT-VALUE})")
   private final Long rpcMaxLogsRange = 5000L;
 
+  @CommandLine.Option(
+      names = {"--cache-last-blocks"},
```

Comment:
this new option needs to be added to the "everything toml" in `./besu/src/test/resources/everything_config.toml`